app/containers/message/Components/RightIcons/ReadReceipt.tsx

```diff
@@ -6,15 +6,13 @@ import { useTheme } from '../../../../theme';
 
 const ReadReceipt = React.memo(({ isReadReceiptEnabled, unread }: { isReadReceiptEnabled?: boolean; unread?: boolean }) => {
 	const { colors } = useTheme();
+	const iconName = unread ? 'check' : 'check-double';
```

Comment:
What about `check-single` Joe mentioned?
It would keep the same size all the time on this component.